presidio-anonymizer/presidio_anonymizer/entities/engine/pii_entity.py

```diff
@@ -0,0 +1,43 @@
+import logging
+from abc import ABC
+
+from presidio_anonymizer.entities import InvalidParamException
+from presidio_anonymizer.services.validators import validate_parameter_not_empty, \
+    validate_parameter_exists, validate_type
+
+
+class PIIEntity(ABC):
+    """Abstract class to hold the text we are going to operate on metadata."""
+
+    def __init__(self, start: int, end: int, entity_type: str):
+        self.logger = logging.getLogger("presidio-anonymizer")
```

Comment:
same comment about logger.

Reply:
It is being used by the inherited classes...